src/valgrind_ctime_test.c

```diff
@@ -25,16 +25,49 @@
 #include "include/secp256k1_schnorrsig.h"
 #endif
 
+void run_tests(secp256k1_context *ctx, unsigned char *key);
+
 int main(void) {
     secp256k1_context* ctx;
+    unsigned char key[32];
+    int ret, i;
+
+    if (!RUNNING_ON_VALGRIND) {
+        fprintf(stderr, "This test can only usefully be run inside valgrind.\n");
+        fprintf(stderr, "Usage: libtool --mode=execute valgrind ./valgrind_ctime_test\n");
+        exit(1);
```

Comment:
Since you're touching this anyway, can you `#include <stdlib.h>` (or simply `return 1` here)?

Reply:
Oh and we also need `stdio` for `fprintf`.

Reply:
done